Skeleton/include/Volunteer.h

```diff
@@ -77,10 +81,11 @@ class DriverVolunteer: public Volunteer {
     public:
         DriverVolunteer(int id, const string &name, int maxDistance, int distancePerStep);
         DriverVolunteer *clone() const override;
-
+        //DriverVolunteer(const DriverVolunteer &other);
         int getDistanceLeft() const;
         int getMaxDistance() const;
         int getDistancePerStep() const;  
+        void setDistanceLeft(int distanceLeft);
```

Comment:
if its only for internal use, put it in private

Reply:
I think it's not possible because LimitedDriverVolunteer cannot inherit private functions from DriverVolunteer, maybe we should use it as protected?

Reply:
good point might be, public or protected you choose